### ISSUE TITLE
Passing an array for testFiles in cli is parsed as a string instead of an array

### ISSUE THREAD

**Issue by jerryhaolin** [cypress-io/cypress#7288]
### Current behavior:
- Passing an array for testFiles and ignoreTestFiles in cli is parsed as a string instead of an array in the cypress runner. This causes the argument to not work.

Command used taken directly from documentation : 
https://docs.cypress.io/guides/references/configuration.html#Command-Line
cypress open --config watchForFileChanges=false,testFiles=["**/*.js","**/*.coffee"]

screenshot from Configuration:
<img width="229" alt="Screen Shot 2020-05-09 at 6 34 07 PM" src="https://user-images.githubusercontent.com/44203435/81486590-b23bde00-9223-11ea-9892-e038cfc7a054.png">

### Desired behavior:

testFiles should be parsed as an array

### Versions

Cypress version 4.5.0

**Comment by jennifer-shehane** [comment]
I can't even get this command to run from the docs. 

<img width="686" alt="Screen Shot 2020-05-12 at 1 15 47 PM" src="https://user-images.githubusercontent.com/1271364/81647278-c285d000-9452-11ea-9331-53e1821742e3.png">

What operating system are you using?

**Comment by jerryhaolin** [comment]
Hi Jennifer thanks for the quick response! I am using macOS 10.14.6. It seems like an array does not work when entered through command line but works when it is in the env file or a config file

**Comment by jennifer-shehane** [comment]
Yes, I think that is the case - the CLI args don't parse the arguments correctly. A workaround will be added the `testFiles` within the configuration file.

**Comment by jerryhaolin** [comment]
That would work but it would not be ideal to have to manually edit the configuration file or env file every time. Especially when the selected tests need to be run from cli (in CI) and need to specify them with an array.

**Comment by jennifer-shehane** [comment]
Alright so you basically need to pass a JSON.stringified object with single quotes around it in order to get this to work. I know, it's super pretty 😅 

```
cypress open --config '{"watchForFileChanges":false,"testFiles":["**/*.js","**/*.coffee"]}' 
```

I'm going to transfer this issue to our documentation repo since this is purely a documentation issue and not an issue in the product itself.
